Makefile

```diff
@@ -115,8 +116,8 @@ ifneq ($(TRAVIS), false)
 	# We're on Travis, so we can lint static files locally
 	./node_modules/.bin/eslint 'warehouse/static/js/**' '**.js' 'tests/frontend/**' --ignore-pattern 'warehouse/static/js/vendor/**'
 	./node_modules/.bin/sass-lint --verbose
-else
-	# We're not on Travis, so we should lint static files inside the static container
+else ifneq ($(ACTIONS), true)
+	# We're not on Travis or GitHub Actions, so we should lint static files inside the static container
```

Comment:
We should do the same thing as if we were on Travis here. As-is, this is not running any static linting on Actions.